docs/news.d/ice.rst

```diff
@@ -0,0 +1,6 @@
+.. news-prs: 4230
+
+.. news-start-section: Fixes
+- ICE doesn't use IPV6-map IPV4 addresses.
```

Comment:
```suggestion
- ICE doesn't use IPv6-mapped IPv4 addresses anymore.
```

The ways it's currently phrased as a fact without any relation to how it worked before sounded a little weird to me.